LayoutTests/fast/canvas/webgl/colorspaces-test.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>WebGL Canvas Conformance Tests</title>
```

Comment:
Perhaps change the title if this doesn't originate from the WebGL conformance test suite to avoid confusion.